Dockerfile

```diff
@@ -1,10 +1,14 @@
 # ================================================================
 #
 #   Dockerfile for RELI (public release)
-#
```

Comment:
Misha, what's the purpose of this (the `Version: 0.0.2`)? I like the idea of having examples for "Build," "Pull," and "Run," but I'm trying to understand why the version tag can't always just be 'latest'.

Reply:
Btw, try to rebuild Dockerfile. I guess, it should me much smaller, then the original one.